### ISSUE TITLE
OmegaDiffusivityMeasures of segmented trajectories: crashes easily

### ISSUE THREAD

**Issue by strambc** [OmegaProject/Omega#166]
When selecting segmented trajectories after an initial period in which everything worked, the app apparently crashed and selection was then impossible and erratic.
Nothing apparently on log.

**Comment by strambc** [comment]
Problem appears to be connected with selecting different segmentation runs or going back to unsegmented.
The TDM module does not appear to be able to refresh to the correct segmentation, so the same graphs are displayed no matter what the selection is.
One clue might be that the Tracking Measures run does not appear to take new information after the first run has been executed.
See for example the attached screen shots.
In other words if one segments --> runs analysis on segments --> dispaly --> wants to take a look at the unsegmented --> the browser does not display any analysis run specific result for the default segmentation. Same happens if one produces a new segmentation run

**Comment by strambc** [comment]
![data selection 0](https://cloud.githubusercontent.com/assets/4050585/23467538/52396b7e-fe6c-11e6-91e7-3205197985c0.png)
![data selection 1](https://cloud.githubusercontent.com/assets/4050585/23467540/523bcc2a-fe6c-11e6-890b-31715004859d.png)
![data selection 2](https://cloud.githubusercontent.com/assets/4050585/23467539/5239fa80-fe6c-11e6-9cb3-2865bf33f48b.png)

**Comment by Akramhar** [comment]
seems to be fixed with the new version
